src/quo/components/notifications/toast/style.cljs

```diff
@@ -21,13 +21,13 @@
 
 (defn content-container
   [theme]
-  {:background-color (colors/theme-colors colors/neutral-80-opa-70 colors/white-opa-70 theme)
-   :flex-direction   :row
-   :justify-content  :space-between
-   :padding-vertical 8
-   :padding-left     10
-   :padding-right    8
-   :border-radius    12})
+  {:background-color   (colors/theme-colors colors/neutral-80-opa-70 colors/white-opa-70 theme)
+   :flex-direction     :row
+   :align-items        :center
+   :justify-content    :space-between
+   :padding-vertical   8
+   :padding-horizontal 8
+   :border-radius      12})
```

Comment:
This happends before.
I guess what designer wants is align when text is single line and do not align when there're multiple lines of text.

Reply:
this breaks multiline alignment

![CleanShot 2024-01-12 at 09 44 05](https://github.com/status-im/status-mobile/assets/15090582/7e04b1f8-60e7-4ab5-9e5b-8e75ea9e4127)
![CleanShot 2024-01-12 at 09 47 47@2x](https://github.com/status-im/status-mobile/assets/15090582/927a7bb4-10a6-4aac-b371-202aac993680)